src/import/load-costume.js

```diff
@@ -8,8 +8,7 @@ const loadVector_ = function (costume, runtime, rotationCenter, optVersion) {
         if (optVersion && optVersion === 2 && !runtime.v2SvgAdapter) {
             log.error('No V2 SVG adapter present; SVGs may not render correctly.');
         } else if (optVersion && optVersion === 2 && runtime.v2SvgAdapter) {
-            runtime.v2SvgAdapter.loadString(svgString, true /* fromVersion2 */);
-            svgString = runtime.v2SvgAdapter.toString();
+            svgString = runtime.v2SvgAdapter(svgString);
```

Comment:
I'm planning to change this to always run, regardless of whether optVersion === 2 or not, since we want to run fixup on all SVGs. After that, we shouldn't need to export V2SVGAdapter from ScratchSVGRenderer, and we shouldn't need to run fixup in paint (since SVGs will pass through loadCostume before being loaded in paint)

Reply:
Let me know once that change goes in--I can rebase this after it does, to instead call:
```js
serializeSvgToString(loadSvgString(svgString, optVersion === 2 /* fromVersion2 */));
```

One issue is that once we start exporting multiple individual functions, passing them *all* in from the GUI (whereas before, we only had to pass in a single `SVGRenderer` object) could start to get cumbersome. IIRC this change was made to make updating the SVG renderer dependency less painful--is there some way to encode "this package works with any version of `scratch-svg-renderer`, but use the parent package's version"?

Reply:
I think peer dependencies? I would have to research them more.